src/test-e2e/endpoints/endpoints-connect-e2e.spec.ts

```diff
@@ -20,6 +22,8 @@ describe('Endpoints', () => {
   describe('Connect/Disconnect endpoints -', () => {
 
     beforeAll(() => {
+      // Ran independently these tests are fine. However stacked with other spec files they'll fail without a pause
+      browser.sleep(1000);
```

Comment:
I'm not a fan of using sleep unless we know why we're doing it... if we're waiting for something to load, complete or appear, I'd prefer to explicitly wait on that